### PR TITLE
Node health check service

### PR DESCRIPTION
**Pull request by murgatroid99**
This is part of #2181

### CODE REVIEW

**Comment by murgatroid99** [comment]
I updated to match the specification changes.

File: src/node/src/server.js [hunk 1]
```diff
@@ -634,7 +634,8 @@ function makeServerConstructor(service_attr_map) {
         }
         var serialize = attrs.responseSerialize;
         var deserialize = attrs.requestDeserialize;
-        server.register(attrs.path, service_handlers[service_name][name],
+        server.register(attrs.path, _.bind(service_handlers[service_name][name],
+                                           service_handlers[service_name]),
```
Comment (yang-g) [inline src/node/src/server.js]: I do not really know what this is doing :)

**Comment by yang-g** [comment]
LGTM

File: src/node/src/server.js (hunk 1 above)
Comment (murgatroid99) [inline src/node/src/server.js]: It just makes sure that each server method implementation has its `this` bound to the object it was passed in attached to.

**Pull request merged**